Microsoft.Alm.Authentication/Src/TokenType.cs

```diff
@@ -60,22 +60,16 @@ public enum TokenType
         [System.ComponentModel.Description("Test-only Token")]
         Test = 5,
 
-        /// <summary>
-        /// Bitbucket Password Tokens.
-        /// </summary>
-        [System.ComponentModel.Description("Bitbucket Password Token")]
```

Comment:
Effectively the Password enum is never used, we only ever use the enum for the Access Token credentials, so remove the password version and renumber the remaining ones.